packages/jest-haste-map/package.json

```diff
@@ -11,22 +11,29 @@
   "types": "build/index.d.ts",
   "dependencies": {
     "@jest/types": "^24.6.0",
+    "anymatch": "^2.0.0",
```

Comment:
Uses same dependency as already required by `sane`, just adding explicitly since we rely on it now too.